### ISSUE TITLE
Compatibility with python 3.8

### ISSUE THREAD

**Issue by deathaxe** [facelessuser/ExportHtml#94]
The `verbose` argument of `namedtuble` function has been removed in python 3.7

The explicit use of it therefore causes the plugin to fail loading on ST's python 3.8 plugin host.

see: https://github.com/facelessuser/ExportHtml/blob/dad8ce2f9f0c8d5216cc2d6274ff468f36cb39ae/lib/color_scheme_matcher.py#L81

As it defaults to `False` it is safe to be removed from in favor of "forward" compatibility with ST4.

**Comment by facelessuser** [comment]
Yeah, it isn't even needed. I'm not quite sure why I thought it was.

**Comment by facelessuser** [comment]
I actually already have it removed locally as I was prepping the library to use the new `coloraide` version and cleaned up such things.

So this will be removed as I update `mdpopups` with the latest `coloraide` (ExportHtml just uses the vendored `coloraide` in `mdpopups`).
